arch/arm/core/cortex_r/irq_init.c

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2014 Wind River Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ *
+ * @brief Initialize interrupts
+ *
+ * For Cortex R, no need to do anything
+ *
+ * @return N/A
+ */
+
```

Comment:
Does this really need to be in its own C file?
Please just declare as a static inline function in kernel_arch_func.h

Reply:
fixed